<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>jAccountSettingsClass</class>
 <widget class="QWidget" name="jAccountSettingsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>jAccountSettings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Account</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <property name="labelAlignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>JID:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="jidEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Password:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="paswdEdit">
         <property name="inputMask">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Resource:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="resourceEdit">
         <property name="inputMask">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Priority:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QSpinBox" name="prioritySpinBox">
             <property name="minimum">
              <number>-9999</number>
             </property>
             <property name="maximum">
              <number>9999</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="priorityCheck">
           <property name="text">
            <string>Set priority depending of the status</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QCheckBox" name="autologinCheck">
         <property name="text">
          <string>Autoconnect at start</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QCheckBox" name="previousStatusCheck">
         <property name="text">
          <string>Keep previous session status</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QCheckBox" name="localBookmarkCheck">
         <property name="toolTip">
          <string>Use this option for servers doesn't support bookmark</string>
         </property>
         <property name="text">
          <string>Local bookmark storage</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Connection</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Encrypt connection:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QCheckBox" name="compressCheckBox">
         <property name="text">
          <string>Compress traffic (if possible)</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Host:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Port:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLineEdit" name="hostEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QSpinBox" name="portSpinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>5222</number>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>148</width>
           <height>46</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QCheckBox" name="manualCheckBox">
         <property name="text">
          <string>Manually set server host and port</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QComboBox" name="tlsComboBox">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>Never</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>When available</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Always</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Proxy</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="margin">
        <number>4</number>
       </property>
       <item>
        <layout class="QFormLayout" name="formLayout_3">
         <property name="labelAlignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Proxy type:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="proxyTypeComboBox">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>HTTP</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>SOCKS 5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Default</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Host:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="proxyHostEdit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Port:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="proxyPortSpinBox">
           <property name="maximum">
            <number>65535</number>
           </property>
           <property name="value">
            <number>80</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="proxyAuthBox">
         <property name="text">
          <string>Authentication</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_4">
         <property name="labelAlignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="userNameLabel">
           <property name="text">
            <string>User name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="proxyUserEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="userPasswordLabel">
           <property name="text">
            <string>Password:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="proxyPasswordEdit">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>137</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>318</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>paswdEdit</tabstop>
  <tabstop>autologinCheck</tabstop>
  <tabstop>compressCheckBox</tabstop>
  <tabstop>manualCheckBox</tabstop>
  <tabstop>hostEdit</tabstop>
  <tabstop>portSpinBox</tabstop>
  <tabstop>proxyTypeComboBox</tabstop>
  <tabstop>proxyHostEdit</tabstop>
  <tabstop>proxyPortSpinBox</tabstop>
  <tabstop>proxyAuthBox</tabstop>
  <tabstop>proxyUserEdit</tabstop>
  <tabstop>proxyPasswordEdit</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>applyButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>jAccountSettingsClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>513</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>jAccountSettingsClass</receiver>
   <slot>okButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>513</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>applyButton</sender>
   <signal>clicked()</signal>
   <receiver>jAccountSettingsClass</receiver>
   <slot>applyButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>443</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>516</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manualCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>hostEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manualCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>portSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>proxyAuthBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>proxyUserEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>proxyAuthBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>proxyPasswordEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>priorityCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>prioritySpinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autologinCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>previousStatusCheck</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>okButtonClicked()</slot>
  <slot>applyButtonClicked()</slot>
 </slots>
</ui>
